src/main.py

```diff
@@ -284,10 +284,8 @@ def _handle_result(self, result):
             self._play_assistant_response(result.response_audio)
         elif result.transcript:
             logger.warning('%r was not handled', result.transcript)
-            self.say(_("I don’t know how to answer that."))
```

Comment:
For reference, this happens in these cases:
- Cloud Speech: failed to recognize command
- Cloud Speech: false positive trigger followed by some speech not intended as a command
- Assistant has handled the command silently, eg 'set volume to 6'.

The currently behaviour is only useful in the first case, and annoying in both others.